api/package.json

```diff
@@ -43,6 +43,8 @@
     "prosemirror-menu": "^1.0.0",
     "prosemirror-state": "^1.0.0",
     "prosemirror-view": "^1.0.5",
+    "repeat-string": "^1.6.1",
+    "romanize": "^1.1.1",
```

Comment:
Sometimes I wish JSON files supported comments, so we could add a little bit of context briefly explaining what a module does and why we need it.

Also, `repeat-string` makes me wish JS supported `"some string" * 5` like Python does.